command/credits.go

```diff
@@ -41,19 +41,22 @@ func init() {
 }
 
 var creditsCmd = &cobra.Command{
-	Use:   "credits [repository]",
-	Short: "View project's credits",
-	Long: `View animated credits for this or another project.
-
-Examples:
-
-  gh credits            # see a credits animation for this project
+	Use:   "credits",
+	Short: "View credits for this tool",
+	Long:  `View animated credits for gh, the tool you are currently using :)`,
+	Example: `gh credits            # see a credits animation for this project
   gh credits owner/repo # see a credits animation for owner/repo
   gh credits -s         # display a non-animated thank you
   gh credits | cat      # just print the contributors, one per line
 `,
-	Args: cobra.MaximumNArgs(1),
-	RunE: credits,
+	Args:        cobra.ExactArgs(0),
+	RunE:        ghCredits,
+	Annotations: map[string]string{"hidden": "true"},
```

Comment:
What is the difference between setting this vs. the `Command.Hidden` property?

Reply:
the difference is that I didn't know about `Command.Hidden` :facepalm: